light-system-programs/programs/merkle_tree_program/src/lib.rs

```diff
@@ -108,6 +118,12 @@ pub mod merkle_tree_program {
         ctx: Context<InitializeMerkleTreeAuthority>,
     ) -> Result<()> {
         ctx.accounts.merkle_tree_authority_pda.pubkey = ctx.accounts.authority.key();
+
+        let event_merkle_tree = &mut ctx.accounts.event_merkle_tree.load_init()?;
+        let merkle_tree_authority = &mut ctx.accounts.merkle_tree_authority_pda;
+
+        process_initialize_new_event_merkle_tree(event_merkle_tree, merkle_tree_authority);
```

Comment:
What do you think about creating the event Merkle tree in the method newTransactionMerkleTree as well?
(Then we don't need to do two transactions every time we create a new TransactionMerkleTree.)

Reply:
I like the idea, will do that.

Reply:
Is this implemented?

Reply:
Not yet, I forgot about it. I will do in the next push.

Reply:
Done